.travis.yml

```diff
@@ -30,10 +31,21 @@ perl:
 #     on_failure: always
 env:
   global:
-    - AUTHOR_TESTING=1
     - AUTOMATED_TESTING=1
-    - RELEASE_TESTING=0
+matrix:
+  include:
+    - perl: stable
```

Comment:
This is redundant; 5.28 is already in the test matrix.  We shouldn't allow failures for 5.28.

Reply:
Incorrect.

As configured, "stable" and "5.28" must pass, excepting that "stable" when combined with "AUTHOR_TESTING=1" and "RELEASE_TESTING=1" is allowed to fail (which *it always does* as I noted previously; see "Allowed Failures" in <https://travis-ci.org/rivy/perl.File-Temp/builds/494642574>).

Reply:
> the current stable perl version (v5.18.2 as of now)

This is defined by travis?  I don't know why travis's idea of "stable" is one that we should be treating as canonical.  5.18 was released several years ago.

Reply:
`perlbrew` seems to be what `build-perl` (from the travis helpers) is using to install `perl`. `perlbrew` is supposed to understand "stable", "blead", and "dev". But, in looking at the travis output, it looks like all of those are unrecognized and leaving perl in the default state (v5.18). It's likely an older version of `perlbrew` though I'm not sure whether it's on the Travis CI or "travis helper" side. If that's updated, this current formulation would be correct. Until then, simple versions are all that seem to work.

Ultimately, I didn't have an desire to touch the Travis CI config, and I'm happy to rip it completely out. But I have never been able to get any commit to pass Travis CI testing (as I noted above).

Reply:
Looking closer at the Travis CI output... the initial more prominently displayed `perl` version is not the final version which is used. After the `build-perl` helper completes, the perl versions used, as of 2019-02-19, are: "stable" => 5.28.1, "blead" => 5.29.8, "dev" => 5.29.7.

So, my initial statements about the matrix and "Allowed failures" hold true.

Per your wishes, I'll pull the AUTHOR_TESTING/RELEASE_TESTING variation if I can get it to test successfully.